### PR TITLE
testing/darktable: new aport

### PR DESCRIPTION
**Pull request by Ikke**
Musl support was recently added, so there is no tag to use yet.

### CODE REVIEW

**Comment by Ikke** [comment]
darktable fails to build because it relies on one of the packages in this commit

**Comment by jirutka** [comment]
Ahaa, this is not a Travis issue! darktable depends on pugixml-dev, but pugixml abuild does not provide -dev subpackage. That’s why it failed.

File: testing/darktable/APKBUILD [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+builddir="$srcdir/$pkgname-$_commit"
+
+prepare() {
+	cd "$builddir"
+	mkdir -p build
+}
+
+build() {
+	cd "$builddir/build"
+	cmake -DCMAKE_INSTALL_PREFIX=/usr .. || exit 1
```
Comment (jirutka) [inline testing/darktable/APKBUILD]: APKBUILD should never `exit` the abuild, there should be `return 1` instead.

File: testing/pugixml/APKBUILD [hunk 2]
```diff
@@ -0,0 +1,27 @@
...
+url="http://pugixml.org"
+arch="all"
+license="MIT"
+makedepends="cmake"
+source="http://github.com/zeux/pugixml/releases/download/v1.7/pugixml-$pkgver.tar.gz"
+builddir="$srcdir/$pkgname-$pkgver"
+
+build() {
+	cd "$builddir"
+	cmake -DCMAKE_INSTALL_PREFIX=/usr -DBUILD_SHARED_LIBS=YesPlease scripts/ || exit 1
```
Comment (jirutka) [inline testing/pugixml/APKBUILD]: APKBUILD should never `exit` the abuild, there should be `return 1` instead.

File: testing/pugixml/APKBUILD [hunk 3]
```diff
@@ -0,0 +1,27 @@
+# Contributor: Kevin Daudt <ops@ikke.info>
+# Maintainer: Kevin Daudt <ops@ikke.info>
+pkgname=pugixml
+pkgver=1.7
+pkgrel=0
+pkgdesc="Light-weight, simple and fast XML parser for C++ with XPath support"
+url="http://pugixml.org"
+arch="all"
+license="MIT"
+makedepends="cmake"
```
Comment (jirutka) [inline testing/pugixml/APKBUILD]: You need to add `subpackages="$pkgname-dev"` here.

File: testing/darktable/APKBUILD [hunk 4]
```diff
@@ -0,0 +1,39 @@
...
+# Maintainer: Kevin Daudt <ops@ikke.info>
+_commit=de8ac28bbb45db1b3eb9b0d51d5ec1288cdfc9dc
+pkgname=darktable
+pkgver=2.1.0.1928
+pkgrel=1
+pkgdesc="an open source photography workflow application and raw developer"
+url="http://www.darktable.org/"
+arch="all"
+license="GPL3"
+depends="libxslt gtk+3.0-dev libgphoto2-dev libwebp-dev
```
Comment (jirutka) [inline testing/darktable/APKBUILD]: This doesn’t look right. Why darktable depends on -dev packages (i.e. header files and similar) in runtime?

**Pull request closed without merging**